ts41/snake-cased-properties.d.ts

```diff
@@ -0,0 +1,23 @@
+import {DelimiterCasedProperties} from './delimiter-cased-properties';
+
+/**
+Convert object props to snake_case but not recursively.
+
+This can be useful when, for example, converting some api type from other style.
+
+@see SnakeCase
+@see SnakeCasedPropertiesDeep
+@example
```

Comment:
```suggestion
@example

```

Reply:
`SnakeCased` don't have a newline after `@example`, which style should follow?